src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,8 @@
-const humanCatDogYears = number => {};
+const humanCatDogYears = number => {
+  const catYears = (number - 2) * 4 + 24;
+  const dogYears = (number - 2) * 5 + 24;
+  const array = [number, catYears, dogYears];
+  return array;
```

Comment:
What if number is 1?
catYears equals 20 and dogYears equals 19, but they should both equal to 15, according to the explanation of the exercise.

You might need to reconsider your logic here!